docs/reference/index-modules/merge.asciidoc

```diff
@@ -29,3 +29,5 @@ The merge scheduler supports the following _dynamic_ setting:
     works well for a good solid-state-disk (SSD).  If your index is on spinning
     platter drives instead, decrease this to 1.
 
+NOTE: `index.merge.scheduler.max_thread_count` does not influence the merge threads
```

Comment:
I'm not sure this is wholly accurate. Force-merging indicates to the shard that a merge is desired and then waits for that merge to complete, but the actual work of the merge will happen on one of the merge threads that is controlled by `index.merge.scheduler.max_thread_count`. I'll ping the Lucene folks for advice here.

Reply:
That's not correct ;). 
When executing a force merge, the merge policy determines what set of merge operations is necessary in order to merge to the specified segment count. The number of segments per merge operation is limited so a force merge that reduces to a single segment can use multiple merges (cascading merge).
These merge operations are then added to the merge queue and executed by one of the merge threads controlled by `index.merge.scheduler.max_thread_count`. So in practice, any force merge can use up to `#max_thread_count` threads.